doc/HLD_of_Auxiliary_Databases_for_SNS_repair.rst

```diff
@@ -0,0 +1,300 @@
+High level design of Auxiliary Databases for SNS repair
```

Comment:
This document is already available at 
https://github.com/Seagate/cortx-motr/blob/main/doc/HLD_of_Auxillary_Databases.rst

Is this the same or similar document?

Reply:
`

> This document is already available at
> https://github.com/Seagate/cortx-motr/blob/main/doc/HLD_of_Auxillary_Databases.rst
> 
> Is this the same or similar document?

Yes, this is the same document. The information is not updated in the following document: https://github.com/Seagate/cortx-motr/blob/main/doc/motr-design-doc-list.rst 

So, I got confused and created the file. But the available file requires few additional changes I will update it. Meanwhile let me remove this file from the PR.